web/api/maestro_api/controllers/run_plan.py

```diff
@@ -47,20 +49,25 @@ def all(self, user):
 
         return jsonify_list_of_docs(run_plans)
 
-    def download(self, run_plan_id, user):
+    def download(self, run_plan_id, user, data):
```

Comment:
Can you add `validation_schema` for this endpoint and have the original_plan value with two options: `true`, `false`. It can be done as Enum or another way with the idea in mind that we can accept only two options here.

We could use the same pattern for all other endpoints where we need bool as query param